src/query/functions/temporal/rate.go

```diff
@@ -131,8 +136,8 @@ func (r *rateNode) process(datapoints ts.Datapoints, bounds iterationBounds) flo
 func standardRateFunc(
 	datapoints ts.Datapoints,
 	isRate, isCounter bool,
-	rangeStart int64,
-	rangeEnd int64,
+	rangeStart xtime.UnixNano,
+	rangeEnd xtime.UnixNano,
```

Comment:
nit: Same nit here, don't need to repeat types, e.g. `rangeStart, rangeEnd xtime.UnixNano`

Reply:
Yeah same as above